internal/services/storage/storage_account_data_source.go

```diff
@@ -155,6 +156,11 @@ func storageAccountDataSource() *schema.Resource {
 			},
 
 			"tags": tags.SchemaDataSource(),
+
+			"enable_https_traffic_only": {
```

Comment:
we always put enabled on the end of the property
```suggestion
			"https_traffic_only_enabled": {
```
or
```suggestion
			"only_https_traffic_allowed_enabled": {
```

Reply:
Done :D